### ISSUE TITLE
Why, after downloading the database snapshot, do I see old blocks in the logs?

### ISSUE THREAD

**Issue by pefka** [bnb-chain/bsc#1048]
Hi all,
I downloaded a snapshot of the database, it's a few days old.
Started geth, then stopped. Then, following the instructions, I copied the new data to the node/geth folder

And now I look at the logs and see, and I have the feeling that it didn’t see this snapshot there and is trying to index from scratch?
2022-08-12T17:56:54+0200 lvl=info msg="Imported new block receipts"           count=932  elapsed=603.524ms number=1,703,647  hash=0xa4c69ef89f7e084ccd7652cf920006d248ad37952bc7020939e82d2802944687 **age=1y9mo3w**   size="2.42 MiB"
t=2022-08-12T17:56:54+0200 lvl=info msg="Imported new block headers"            count=2048 elapsed=232.964ms number=1,744,464  hash=0xed07a2e31fd97f0ac863bec5bdc8e1706b2c00b55739eeebe46c27864f687b3e **age=1y9mo3w**

How long did you have to wait for synchronization after you started the node from the snapshot?

**Comment by j75689** [comment]
I think you may not have specified the correct `--datadir` directory.

if use the latest snapshot data, It should be able to sync within three days.

**Comment by pefka** [comment]
I did some research and realized that this is normal behavior. You will still see the old blocks in the logs, even if you synchronize the database from the snapshot. And you still need to wait until the system processes even old data. I don’t know how long to wait, I checked after about 12 hours and saw already new blocks in the logs.
It's strange, of course, why not take a snapshot of the entire folder with the geth application and so that when you start it, everything works for you right away.
So the problem is solved.
